core/src/trezor/ui/layouts/mercury/__init__.py

```diff
@@ -1066,36 +1074,36 @@ async def confirm_ethereum_staking_tx(
         br_name: str = "confirm_ethereum_staking_tx",
         br_code: ButtonRequestType = ButtonRequestType.SignTx,
     ) -> None:
-
-        # intro
-        await confirm_value(
-            title,
-            intro_question,
-            "",
-            br_name,
-            br_code,
-            verb=verb,
-            value_text_mono=False,
-            info_items=(("", address),),
-            info_title=address_title,
-            chunkify_info=chunkify,
-        )
-
-        # confirmation
         if verb == TR.ethereum__staking_claim:
-            items = ((TR.send__maximum_fee, maximum_fee),)
+            summary_items = ((TR.send__maximum_fee, maximum_fee),)
         else:
-            items = (
+            summary_items = (
                 (TR.words__amount, total_amount),
                 (TR.send__maximum_fee, maximum_fee),
             )
-        await _confirm_summary(
-            items=items,
-            title=title,
-            info_title=TR.confirm_total__title_fee,
-            info_items=info_items,
-            br_name=br_name,
-            br_code=br_code,
+        await raise_if_not_confirmed(
+            RustLayout(
+                trezorui2.flow_confirm_output(
+                    title=verb,
+                    subtitle=None,
+                    message=intro_question,
+                    amount=None,
+                    chunkify=False,
```

Comment:
Why is `chunkify` set to `False`?

Reply:
Ok, I wasn't aware that we *always* chunkify in `mercury`.

Reply:
> Ok, I wasn't aware that we *always* chunkify in `mercury`.

I mean, we always chunkify the address on the (separate) address screen in mercury, because this is the only place we have that screen (it is the one I added now in this PR, so it is only here).